src/theory/ff/theory_ff_rewriter.h

```diff
@@ -67,6 +67,28 @@ class TheoryFiniteFieldsRewriter : public TheoryRewriter
   RewriteResponse preRewrite(TNode node) override;
 }; /* class TheoryFiniteFieldsRewriter */
 
+/** preRewrite negation */
+Node preRewriteFfNeg(TNode t);
+
+/** preRewrite addition */
+Node preRewriteFfAdd(TNode t);
+/** postRewrite addition */
+Node postRewriteFfAdd(TNode t);
+
+/** preRewrite multiplication */
+Node preRewriteFfMult(TNode t);
+/** postRewrite multiplication */
+Node postRewriteFfMult(TNode t);
+
+/** postRewrite equality */
+Node postRewriteFfEq(TNode t);
+
+/** Parse as a product with a constant scalar */
+std::optional<std::pair<Node, FfVal>> parseScalar(TNode t);
+
+/** Make an n-ary operator of length is more than 1 */
+Node mkNary(Kind k, std::vector<Node>&& children);
```

Comment:
If those helper methods are not used anywhere else, I'd recommend not declaring them in the header file, but just to have them in an anonymous namespace in the source file. They'll only be linked internally that way.